pycolmap/custom_incremental_mapping.py

```diff
@@ -0,0 +1,309 @@
+import shutil
```

Comment:
Maybe we also call this "example_incremental_mapping.py" and add a brief `""" Comment """` to the header explaining what the purpose of this example is (i.e., equivalent logic as C++ incremental mapper, etc.)?

Reply:
I have added a draft description. Since Paul has restructured the file a bit and only use ``example.py`` as the main runner, I am looping him in here @sarlinpe  for potential further changes on this : )

Reply:
Thanks, this looks good.